Engines/Wine/Engine/Implementation/script.js

```diff
@@ -335,13 +347,9 @@ var engineImplementation = {
         this.install(subCategory, version);
 
         var wineBinary = this.getLocalDirectory(subCategory, version) + "/bin/wine";
-        var StringArray = Java.type('java.lang.String[]');
-        var command = new StringArray(2 + args.length);
-        command[0] = wineBinary;
-        command[1] = executable;
-        java.lang.System.arraycopy(args, 0, command, 2, args.length);
```

Comment:
@plata I think the problem you encountered previously was because of this line. 
The issue was that `args` is a JavaScript list, while `command` is a Java array. `System#arraycopy` doesn't support that signature, therefore the error.

I fixed that problem, but I think we should remove all direct calls to Java APIs like `ProcessBuilder`, to be sure to prevent the similar issues from occurring.

Reply:
Agree. Also @qparis mentioned this before. Can you open an issue to track that?